docs/development.md

```diff
@@ -8,6 +8,37 @@ The source files are in the `/source` directory.  The `compile` rake task builds
 
 This resulting app directory is included in the gem and hooked in as a Rails engine
 
+## Getting a manual test rig setup
```

Comment:
I believe there's a rails app to do this instead?

Not sure how http://alphagov.github.io/govuk_template/ is served but I imagine this could be used to test changes.